api/app-node/documentloader/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2024 KubeAGI.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha1 contains API Schema definitions for the arcadia v1alpha1 API group
+// +kubebuilder:object:generate=true
+// +groupName=arcadia.kubeagi.k8s.com.cn
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+const (
+	Group   = "arcadia.kubeagi.k8s.com.cn"
```

Comment:
Need to double check that since we are using group name `arcadia.kubeagi.k8s.com.cn`, should we move these files to `api/base`?

Reply:
I think about this before, but documentloader is a Node that should run with app runtime, so place it to app-node.
api/base seems just resources that does not have any Node run logic, so don't use it although they have the same group. Let's keep it for now?